frontend/src/scenes/Login.js

```diff
@@ -90,6 +94,15 @@ export default function Login({ locale, routePath }) {
         >
           {STRINGS[locale].FACEBOOK_SIGN_UP}
         </FacebookButton>
+        <Link
```

Comment:
I think glen meant to use
```
<a href="https://m.me/callhomesg" style="...">the icon and text</a>
```
'<Link>' is used for linking to react router routes within the app. `<a>` uses normal html sementics

Reply:
I'm somewhat confident that <Link> behaves like a <a> for routes that live outside of the application. I can test the changes to confirm this.

Reply:
Link works just fine. Let's standardise on using <Link> so that we don't have to worry about choosing <a> or <Link>